lib/utils.dart

```diff
@@ -34,27 +31,31 @@ readData() async {
     features = loadJSON(localFallbacks['feature-config']);
   }
   */
-  Map<String, dynamic> features = await readJSON(localFallbacks["feature-config"]);
+  Map<String, dynamic> features =
+      await readJSON(localFallbacks["feature-config"]);
 
-  Map<String, dynamic> labelsConfig = await readJSON('assets/labels.conf'); // TODO (far out): gather this info not from file but thru GUI input
+  // load labels
+  Map<String, dynamic> labelsConfig = await readJSON(
+      'assets/labels.conf'); // TODO (far out): gather this info not from file but thru GUI input
   Map<String, dynamic> labels = labelsConfig["label_titles"];
   String labelsJSON = jsonEncode({"labels": labelsConfig["labels"]});
 
   // load model coefficients and means
   Map<String, dynamic> models;
   try {
-    http.Response modelsResponse = await http.post(serverAddress + '/models',
-        headers: {"Content-Type": "application/json"},
-        body: labelsJSON)
+    http.Response modelsResponse = await http
+        .post(serverAddress + '/models',
+            headers: {"Content-Type": "application/json"}, body: labelsJSON)
         .timeout(const Duration(seconds: 1));
     models = jsonDecode(modelsResponse.body);
-  }
-  catch (e) { // something with the web request went wrong, use local file fallback
+  } catch (e) {
+    // something with the web request went wrong, use local file fallback
     models = await readJSON(localFallbacks["models"]);
   }
   return [models, features, labels];
 }
 
+/// Sets first radio button or slider to min for all features in [featureConfig].
```

Comment:
Why first?

Reply:
Because there are multiple radio buttons that you could click and that function sets the firstof them, right? Because it sets the value to 0?

Reply:
Oh, I now understand what the sentence means.
```suggestion
/// For all features in [featureConfig]: Select first radio button or set slider to min.
```

Still not 100% on this but the concept of "setting a button" sounded like UI stuff. So select.